### PR TITLE
solved issue of Navbar responsiveness

### PR DESCRIPTION
**Pull request by Mudassirkhan2**
solved the issue of navbar responsiveness by removing github logo for small devices and decreasing padding-x for button get started  
Screenshot 2023-04-23 130933

### CODE REVIEW

**Review by Deepu178 — CHANGES_REQUESTED** [review]
Actuall, you installed dependencies using npm but we have recently shifted to yarn package manager. add the dependencies using yarn.

File: yarn.lock [hunk 1]
```diff
@@ -4,26 +4,26 @@
 
 "@algolia/cache-browser-local-storage@4.17.0":
   version "4.17.0"
-  resolved "https://registry.yarnpkg.com/@algolia/cache-browser-local-storage/-/cache-browser-local-storage-4.17.0.tgz#4c54a9b1795dcc1cd9f9533144f7df3057984d39"
```
Comment (Chandraprakash-Darji) [inline yarn.lock:7]: undo this file changes

File: src/components/Buttons/Button.tsx [hunk 2]
```diff
@@ -34,11 +34,11 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
         ref={ref}
         {...rest}
         className={clsxm(
-          'inline-flex items-center rounded-full',
```
Comment (Chandraprakash-Darji) [inline src/components/Buttons/Button.tsx:37]: don't change button style, instead where button is used, just pass there classname,

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

**Comment by Mudassirkhan2** [comment]
#88

**Pull request closed without merging**